src/eredis_client.erl

```diff
@@ -341,7 +341,10 @@ get_addr(Hostname) ->
                 {error, _} ->
                     case inet:getaddr(Hostname, inet6) of
                         {ok, Addr} -> {ok, {inet6, Addr}};
-                        {error, _} = Res -> Res
+                        {error, _} -> case inet:getaddr(Hostname, inet) of
```

Comment:
I think the fix should be here:
https://github.com/tigertext/eredis/pull/5/files#diff-560becbeb637c5602be2823b68a6fcd1R340

inet4 -> inet